src/components/footer/Footer.tsx

```diff
@@ -995,6 +1008,18 @@ export default function Footer({
                       >
                         {textContent.FooterSection.sections.tools.fileVirusScan}
                       </a>
+                      <a
+                        href="https://internxt.com/password-generator"
```

Comment:
You will have a hard time if some day we change a bunch of URLs at once, a urls.ts file will save your day, just some random advice